tailwind.config.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
           green: "#00C180",
           logo: "#31CE9F",
         },
+        blur_opacity: {
+          one: 'rgba(255, 255, 255, 0.8)',
+          two: 'rgba(255, 255, 255, 0.6)',
+          three: 'rgba(255, 255, 255, 0.4)',
+        }
```

Comment:
I dont think this is the way to do it

Reply:
removed that part! Thank you for letting me know, I thought I discarded it earlier!